clients/client.ts

```diff
@@ -5,8 +5,9 @@ import {
     AminoTypes
 } from "@cosmjs/stargate";
 import { SigningCosmWasmClient, SigningCosmWasmClientOptions } from "@cosmjs/cosmwasm-stargate"
-import { Coin, MsgSend, MessageOptions, Balance, BroadcastAPI, AllAccountsResponse } from "../src"; // Replace with your own Msg import
+import { getSigningCosmosClient } from ".././src/cosmos/client"; // Replace with your own Msg import
```

Comment:
```suggestion
import { getSigningCosmosClient } from "../src/cosmos/client"; // Replace with your own Msg import
```

Reply:
@vishrutsingh this is the last comment, if it looks good, just commit it, then can merge